flake.nix

```diff
@@ -92,6 +97,7 @@
             cargoTestOptions = f: f ++ [ "--all" ];
 
             NIX_INSTALLER_TARBALL_PATH = nixTarballs.${final.stdenv.system};
+            DETERMINATE_NIX_BINARY_PATH = if final.stdenv.system == "x86_64-linux" || final.stdenv.system == "aarch64-linux" then "${inputs.dnee.packages.${final.stdenv.system}.default}/bin/determinate-nix-ee" else null;
```

Comment:
Does this actually work? I'd expect Nix to be kinda unhappy about a `null` env var value.......

Reply:
It does actually work!